src/app/modules/main/chat_section/chat_content/messages/view.nim

```diff
@@ -166,6 +167,12 @@ QtObject:
   proc jumpToMessage*(self: View, messageId: string) {.slot.} =
     self.delegate.scrollToMessage(messageId)
 
+  proc setEditModeOnAndScrollToLastMessage*(self: View, pubkey: string) {.slot.} =
+    let lastMessage = self.model.getLastItemFrom(pubKey)
+    if lastMessage != nil: 
```

Comment:
I think it was always reurn something, since we return `Item`, just a Nim quirk. So you can check like this to be safer:
```suggestion
    if lastMessage != nil and lastMessage.id != "": 
```